src/osd/OSD.cc

```diff
@@ -6403,7 +6403,7 @@ void OSD::do_command(Connection *con, ceph_tid_t tid, vector<string>& cmd, buffe
     auto start = ceph::coarse_mono_clock::now();
     store->compact();
     auto end = ceph::coarse_mono_clock::now();
-    auto time_span = chrono::duration_cast<chrono::duration<double>>(end - start);
+    auto time_span = totimespan(end-start);
     dout(1) << "finished manual compaction in "
             << time_span.count()
```

Comment:
I'm pretty sure this will give you a count in nanoseconds. Which is fine if you label it as nanoseconds, though since this is from coarse-mono-clock I'd just duration_cast it to std::chrono::milliseconds.

Reply:
Ah, that was unintentional. I'll add the cast.